### PR TITLE
Improve configurable BBE

### PR DESCRIPTION
**Pull request by HindujaB**
## Purpose
Improve the BBE with recent improvements 

## Goals
N/A

## Approach
N/A

## User stories
N/A

## Release note
N/A

## Documentation
N/A

## Training
N/A

## Certification
N/A

## Marketing
N/A

## Automation tests
N/A

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

## Samples
N/A

## Related PRs
N/A

## Migrations (if applicable)
N/A

## Test environment
ubuntu 20.04, jdk 11.0.8
 
## Learning
N/A

### CODE REVIEW

File: examples/configurable/configurable.description [hunk 1]
```diff
@@ -4,8 +4,12 @@
 // be applied to a module-level variable declaration that contains an initializer.<br/>
 //
 // The values for `configurable` variables can be provided in a file named
-// `configuration.toml` located in the current directory.
+// `Config.toml`. The file location can be set through an environment variable with
+// the name `BALCONFIGFILE`. If an environment variable is not specified, it is located
```
Comment (warunalakshitha) [inline examples/configurable/configurable.description:8]: May be after alpha we might to revisit this name 'BALCONFIGFILE'

File: examples/configurable/configurable.description [hunk 2]
```diff
@@ -4,8 +4,12 @@
...
+// in the current directory.
+//
 // The configuration file must be in the TOML(v0.4) format.
 // Ballerina currently supports configurable variables of types `int`, `float`,
-// `boolean` and `string`. Future versions will add support for `configurable` variables
+// `boolean`, `string`, `decimal` and the arrays of respective types.
+// Future versions will add support for `configurable` variables
```
Comment (warunalakshitha) [inline examples/configurable/configurable.description:14]: Do we need this line?

File: examples/configurable/configurable.out [hunk 3]
```diff
@@ -9,7 +9,7 @@
 # maxPayload = 0.321
 
 # Then, navigate to the directory that contains the `.bal` and
-# `configuration.toml` files, and execute the `ballerina run` command below.
+# `Config.toml` files, and execute the `ballerina run` command below.
 
 ballerina run configurable.bal
```
Comment (warunalakshitha) [inline examples/configurable/configurable.out:14]: Shall we chnage this to 'bal' and retest with new distribution.

File: examples/configurable/configurable.description (hunk 2 above)
Comment (manuranga) [reply examples/configurable/configurable.description:14]: ya, I also think we don't need the 'Future versions' line

**Review by praneesha — APPROVED** [review]

**Comment by rdulmina** [comment]
We need to add a meta tag file to the BBE.

**Pull request merged**